core/src/main/java/com/linecorp/armeria/server/cors/AbstractCorsPolicyBuilder.java

```diff
@@ -53,7 +56,9 @@
  */
 abstract class AbstractCorsPolicyBuilder {
 
-    private final Set<String> origins;
+    private Set<String> origins = Collections.emptySet();
```

Comment:
Let's revert this and assign `origins = Collections.emptySet();` in the constructors.